tools/generate-files

```diff
@@ -46,9 +46,17 @@ mkdir -p swift/Sources/Generated/Protobuf swift/Sources/Generated/Enums
 # Generate coins info from registry.json
 codegen/bin/coins
 
-# Generate interface code
+# Generate interface code, Swift bindings excluded.
 codegen/bin/codegen
 
+# Generate Swift bindings with codegen-v2. This is a transitionary process 
+# and will eventually deprecate the current codegen/ entirely.
+cd codegen-v2/
+cargo run -- swift
+cp -R bindings/ ../swift/Sources/Generated/
+cp src/codegen/swift/templates/WalletCore.h ../swift/Sources/Generated/
```

Comment:
Could this be a part of the `cargo run -- swift` command? So in `main.rs`?
What do you think?

Reply:
See https://github.com/trustwallet/wallet-core/issues/3159